### PR TITLE
Fix to fit different urls

### PR DESCRIPTION
**Pull request by nilsbehlen**
some work towards custom otp field text and disabling the credential name

### CODE REVIEW

File: helpers/registry.h [hunk 1]
```diff
@@ -19,29 +19,25 @@ enum CONF_VALUE
 {
 	CONF_SERVER_URL = 0,
 	CONF_LOGIN_TEXT = 1,
+	CONF_OTP_TEXT = 2,
```
Comment (cornelinux) [inline helpers/registry.h]: Do we get problems with existing configurations if we change the order here? Or is it only for internal use?

Comment (nilsbehlen) [reply helpers/registry.h]: No. But the other values needed to changed accordingly. 
Below that enum is the array with the corresponding registry names. 
config.h was also adjusted.

**Comment by nilsbehlen** [comment]
This fixes #10, but not #9 yet, as the registry value is read correctly, but it is not used the way its supposed to be, gotta look into that some more

**Comment by cornelinux** [comment]
OK. Please ping me, when I should take a 2nd look and should merge.

**Comment by nilsbehlen** [comment]
It seems clumsy to discuss the questions here, we should do that tomorrow when we meet in person!

**Pull request merged**